section1/exercises/booleans.rb

```diff
@@ -4,13 +4,14 @@
 # `ruby section1/exercises/booleans.rb``
 
 # EXAMPLE: print to the terminal the result of 1 is equal to 2:
-p 1 === 2
+puts 1 === 2
 
 # EXAMPLE: print to the terminal the result of 7 is greater than 2:
-p 7 > 2
+puts 7 > 2
 
 # YOU DO: print to the terminal the result of "hello" is equal to "Hello":
-
+puts "hello" === "Hello"
 # YOU DO: print to the terminal the result of 3 is not equal to 4:
-
+puts 3 =/ 4
```

Comment:
Have you tried running this line (either by executing the file or in irb?) This is not the correct Ruby way to say '3 does not equal 4'